pkg/kp/rollstore/consul_store.go

```diff
@@ -392,11 +392,7 @@ func (s consulStore) CreateRollingUpdateFromOneMaybeExistingWithLabelSelector(
 
 	var oldRCID rc_fields.ID
 	if len(matches) > 1 {
-		return roll_fields.Update{}, util.Errorf(
-			"Can't create update: old RC selector %s was ambiguous, provided %d matches",
-			oldRCSelector.String(),
-			len(matches),
-		)
+		return roll_fields.Update{}, AmbiguousRCSelector
```

Comment:
> which we can easily use to debug with p2-rctl
agree

It seems unlikely that this number will be anything other than 1 or 2. Is there a case that I'm missing?